app/controllers/users_controller.rb

```diff
@@ -0,0 +1,29 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
+    return if @user
+
+    flash[:danger] = t ".not_found"
+    redirect_to root_path
+  end
+
+  def new
```

Comment:
```suggestion
  // GET: /users/new
  def new
```
Thêm comment router phía trên action để tường mình code hơn